warehouse/admin/templates/admin/projects/detail.html

```diff
@@ -159,8 +165,13 @@ <h3 class="box-title">Set upload limit</h3>
       <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
       <div class="box-body">
         <div class="form-group col-sm-12">
-          <label for="uploadLimit">Upload limit (in bytes)</label>
-          <input type="text" name="upload_limit" class="form-control" id="uploadLimit" rows="3" value="{{ project.upload_limit | default('', True)}}">
+          <label for="uploadLimit">Upload limit (in Megabytes)</label>
+          {% if project.upload_limit %}
+            {% set upload_limit_value = project.upload_limit / ONE_MB %}
```

Comment:
Unlike the display issue, this needs to use the same logic as the python code to go from bytes to MB, so this makes sense to keep this regardless of what happens above.

Reply:
Noted.